lib/sqlalchemy/engine/url.py

```diff
@@ -906,8 +906,7 @@ def _parse_url(name: str) -> URL:
         )
 
 
-def _sqla_url_quote(text: str) -> str:
-    return re.sub(r"[:@/]", lambda m: "%%%X" % ord(m.group(0)), text)
+_sqla_url_quote = quote_plus
```

Comment:
**Michael Bayer** (zzzeek) wrote:

this is wrong because we only use 'unquote' when parsing username /password, not plus

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4996